dlang/plugin-impl/src/main/resources/META-INF/plugin.xml

```diff
@@ -1,7 +1,6 @@
 <idea-plugin xmlns:xi="http://www.w3.org/2001/XInclude" allow-bundled-update="true">
     <id>net.masterthought.dlanguage</id>
     <name>D Language</name>
-    <version>${version}</version>
```

Comment:
again, make sure this works correctly. When it was last removed the release version wasn't in the xml causing the release to be messed up